src/core/inc/runtime.h

```diff
@@ -361,7 +362,12 @@ class Runtime {
 
   uint64_t sys_clock_freq() const { return sys_clock_freq_; }
 
-  void KfdVersion(const HsaVersionInfo& version) { kfd_version.version = version; }
+  void KfdVersion(const HsaVersionInfo& version) {
+    kfd_version.version = version;
+    if (version.KernelInterfaceMajorVersion > 1 ||
```

Comment:
I think we should check for MajorVersion == 1 && MinorVersion >= 14. Any major version other than 1 is unsupported as a major version bump would indicate a break in backwards compatibility. I do not expect that we will ever bump the major version on the KFD API in the upstream driver. But if we do, it'll be for a very different ROCr.

Reply:
Sure. I will update it